Assets/Mirror/Core/NetworkServer.cs

```diff
@@ -1149,18 +1149,47 @@ public static bool ReplacePlayerForConnection(NetworkConnectionToClient conn, Ga
 
         /// <summary>Removes the player object from the connection</summary>
         // destroyServerObject: Indicates whether the server object should be destroyed
+        // Deprecated 2024-06-06
+        [Obsolete("Use RemovePlayerForConnection(NetworkConnectionToClient conn, RemovePlayerOptions removeOptions) instead")]
         public static void RemovePlayerForConnection(NetworkConnectionToClient conn, bool destroyServerObject)
         {
-            if (conn.identity != null)
+            if (destroyServerObject)
+                RemovePlayerForConnection(conn, RemovePlayerOptions.Destroy);
+            else
+                RemovePlayerForConnection(conn, RemovePlayerOptions.Unspawn);
+        }
+
+        public enum RemovePlayerOptions 
+        {
+            /// <summary>Player Object remains on server and clients. Only ownership is removed</summary>
+            DoNothing,
```

Comment:
Maybe name this 'Disown' since it doesn't technically do 'nothing'?

Reply:
Changed to KeepActive